gulpfile.js

```diff
@@ -61,13 +61,32 @@ gulp.task('tslint', function () {
         .pipe(tslint.report('verbose'));
 });
 
+function readArgument(argumentName) {
+    var argumentNameIndex = process.argv.indexOf("--" + argumentName);
+    var argumentValueIndex = argumentNameIndex + 1;
+    return argumentNameIndex > -1 && argumentValueIndex < process.argv.length
+        ? process.argv[argumentValueIndex]
+        : null;
+}
+
 function test() {
+    // Defaults
+    var pattern = "extensionContext";
```

Comment:
I find this initialization a little misleading; the value `"extensionContext"` is never used